operations/pyroscope/helm/pyroscope/templates/services.yaml

```diff
@@ -9,6 +9,13 @@ metadata:
   labels:
     {{- include "pyroscope.labels" . | nindent 4 }}
     app.kubernetes.io/component: {{ $component | quote }}
+    {{- if $cfg.labels }}
+      {{- toYaml $cfg.labels | nindent 4 }}
+    {{- end }}
```

Comment:
Could you please tell more about the use case? I'm asking because I think that `app.kubernetes.io/component: {{ $component | quote }}` identifies the service object unambiguously and additional labels should not be required

Reply:
we had a case where we used a specific label to identify the pod but like you mentioned, with a change we switched to the component label, I'll remove that.